AtlasStarter/Program.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.Collections.Generic;
+using MongoDB.Driver;
+
+namespace AtlasStarter
+{
+    class MainClassNoPrompt
+    {
+        public static void Main(string[] args)
+        {
+            // TODO:
+            // Replace the placeholder connection string below with your
+            // Altas cluster specifics. Be sure it includes
+            // a valid username and password! Note that in a production environment,
+            // you do not want to store your password in plain-text here.
+
+            var mongoUri = "mongodb+srv://<username>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+            // The IMongoClient is the object that defines the connection to our
+            // datastore (Atlas, for example)
+            IMongoClient client;
+
+            // An IMongoCollection defines a connection to a specific MongoDB
+            // collection. Your app may have one or many different IMongoCollection
+            // objects.
+            IMongoCollection<Recipe> collection;
+
+            // Note that you must define the *type* of data stored in the
+            // IMongoCollection. We have created a class called TestDocument at
```

Comment:
Should this be...
```suggestion
            // IMongoCollection. We have created a class called `Recipe` at
```
?